config/hubs/pangeo-181919.cluster.yaml

```
@@ -0,0 +1,77 @@
+name: pangeo-181919
+provider: gcp
+gcp:
+  key: secrets/pangeo-181919.json
+  project: pangeo-181919
+  cluster: pangeo-hubs-cluster
+  zone: us-central1-b
+support:
+  config:
+    grafana:
+      ingress:
+        hosts:
+          - grafana.pangeo.2i2c.cloud
+        tls:
+          - secretName: grafana-tls
+            hosts:
+              - grafana.pangeo.2i2c.cloud
+hubs:
+  - name: coessing
+    domain: coessing.pangeo.2i2c.cloud
+    template: daskhub
+    auth0:
+      connection: google-oauth2
+    config:
+      basehub:
+        jupyterhub:
+          proxy:
+            service:
+              type: LoadBalancer
+            https:
+              enabled: true
+          custom:
+            homepage:
+              templateVars:
+                org:
+                  name: Coastal Ocean Environment Summer School in Ghana
+                  url: https://coessing.org/
+                  logo_url: "https://coessing.files.wordpress.com/2016/08/ghana-logo-21.png?w=262&h=376&zoom=2"
+                designed_by:
+                  name: 2i2c
+                  url: https://2i2c.org
+                operated_by:
+                  name: 2i2c
+                  url: https://2i2c.org
+                funded_by:
+                  name: 2i2c
+                  url: https://2i2c.org
+          hub:
+            config:
+              Authenticator:
+                allowed_users: &coessing_users
+                  - paigemar@umich.edu
+                  - drsarahlgibson@gmail.com
+                  - yuvipanda@gmail.com
+                  - choldgraf@gmail.com
+                  - georgiana.dolocan@gmail.com
+                admin_users: *coessing_users
+          singleuser:
+            image:
+              name: pangeo/pangeo-notebook
+              tag: 2021.07.06
```

```suggestion
              name: quay.io/2i2c/coessing-image
              tag: TBA
```

Waiting for https://github.com/2i2c-org/coessing-image/pull/3 to be merged and the image to be built before adding the tag and merging this one

---

**Update:** Added ref `5a8a765b7ff9` to config for bespoke image